src/main/java/mops/module/controller/ModulbeauftragterController.java

```diff
@@ -44,16 +44,11 @@ public String module(KeycloakAuthenticationToken token, Model model) {
         model.addAttribute("allCategories", Modulkategorie.values());
         model.addAttribute("allModules", modulService.getAllModule());
         model.addAttribute("allVisibleModules", modulService.getAllSichtbareModule());
```

Comment:
Vorschlag:
```
List<Modul> allSichtbareModule = modulService.getAllSichtbareModule();
```

und dann alle Vorkommnisse in dieser Methode von `modulService.getAllSichtbareModule()` mit `allSichtbareModule` ersetzen (Zeile 46, 48, 51)